com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Utility/HDUtils.cs

```diff
@@ -625,11 +629,13 @@ public void ComputePackedMipChainInfo(Vector2Int viewportSize)
 
                     mipLevelOffsets[mipLevel] = mipBegin;
 
-                    textureSize.x = Math.Max(textureSize.x, mipBegin.x + mipSize.x);
-                    textureSize.y = Math.Max(textureSize.y, mipBegin.y + mipSize.y);
+                    hardwareTextureSize.x = Math.Max(hardwareTextureSize.x, mipBegin.x + mipSize.x);
+                    hardwareTextureSize.y = Math.Max(hardwareTextureSize.y, mipBegin.y + mipSize.y);
                 }
                 while ((mipSize.x > 1) || (mipSize.y > 1));
 
+                textureSize = new Vector2Int((int)((float)hardwareTextureSize.x * textureScale.x), (int)((float)hardwareTextureSize.y * textureScale.y));
```

Comment:
Is this rounding via cast matching what the actual texture size ends up being?  I think it should be fine, but I remember seeing some non-obvious rounding going on in the past hw side, so worth maybe check via renderdoc if you haven't? 

(Worth checking several steps of percentage, so that we check both odd and even texture sizes )